### ISSUE TITLE
bypass service worker errors

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2750]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.41.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.41.0...4.41.1) (2022-06-18)

- Release 4.41.1 [`54c94ab`](https://github.com/open-wa/wa-automate-nodejs/commit/54c94abacf89fb4fbd2a5a43cbf60b96882ec8b6)
- 🐛 ensure forward and backwards pptr compatibility #2750 [`2e59a9f`](https://github.com/open-wa/wa-automate-nodejs/commit/2e59a9f4ee21990ec10e120aca8c3ce2d9a732f1)
- 🏷️ fix type imports [`42d22ba`](https://github.com/open-wa/wa-automate-nodejs/commit/42d22bae7bae79c929faab0d516bc876b959c117)
- updated types-only package [`28d3c32`](https://github.com/open-wa/wa-automate-nodejs/commit/28d3c32ca078b9e4aa82fa2d0ae51ca84564daa8)
